change/@fluentui-api-docs-84178d5e-3c51-4d54-bed5-7fd3a97c207d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33498&commitId=e4945feaaff508992542ba5f2a6702e2eb5c93b9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Beak 25.chromium.png|2186| Changed|
|Callout.Bottom center.chromium.png|2146| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Right center.chromium.png|2396| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>